docker/bin/zookeeperTeardown.sh

```diff
@@ -36,10 +36,11 @@ fi
 # "terminationGracePeriodSeconds" before focibly killing the container
 CONN_COUNT=`echo cons | nc localhost 2181 | grep -v "^$" |grep -v "/127.0.0.1:" | wc -l`
 for (( i = 0; i < 36; i++ )); do
-  echo "$CONN_COUNT non-local connections still connected."
-  sleep 5
-  CONN_COUNT=`echo cons | nc localhost 2181 | grep -v "^$" |grep -v "/127.0.0.1:" | wc -l`
+  if [[ "$CONN_COUNT" -gt 0 ]]; then
+    echo "$CONN_COUNT non-local connections still connected."
+    sleep 5
+  fi
```

Comment:
Here we can add an else block which would terminate the loop in case the connection count is 0

Reply:
done